herddb-core/src/main/java/herddb/core/system/SyslogstatusManager.java

```diff
@@ -39,8 +39,8 @@
     private final static Table TABLE = Table
             .builder()
             .name("syslogstatus")
-            .column("tablespace_uuid", ColumnTypes.STRING)
-            .column("nodeid", ColumnTypes.STRING)
+            .column("tablespace_uuid", ColumnTypes.NOTNULL_STRING)
```

Comment:
you have to revert this change as well and the line below

Reply:
Done.